h/views/api_moderation.py

```diff
@@ -19,4 +20,8 @@ def create(context, request):
 
     svc = request.find_service(name='annotation_moderation')
     svc.hide(context.annotation)
+
+    event = events.AnnotationEvent(request, context.annotation.id, 'update')
```

Comment:
Hmmm. Not hmmm at your code, but hmmm at needing to do this any time we update an annotation in a way that might require a reindex. I guess we probably need to find a better way of doing this at some point.

For now, it might be worth adding a short comment just to explain why this event needs to be fired.